src/axom/sidre/core/Group.hpp

```diff
@@ -1156,6 +1156,21 @@ class Group
    */
   bool isEquivalentTo(const Group* other, bool checkName = true) const;
 
+  /*!
+   * \brief Return true if this Group holds items in map format.
+   */
+  bool usingMap()
```

Comment:
Thanks for adding those methods, @nselliott 

1. Those should probably be `const`
2. It may be more consistent withe the rest of the API if they are named as: `isUsingMap()` and `isUsingList()`